src/Dodo.HttpClient.ResiliencePolicies/Defaults.cs

```diff
@@ -6,6 +6,7 @@ public static class Timeout
 		{
 			public const int HttpClientTimeoutInMilliseconds = 10000;
 			public const int TimeoutPerTryInMilliseconds = 2000;
+			public const int TimeoutOverallInMillicesons = 100000;
```

Comment:
Could I ask you to make some minor changes:
1. Let's set TimeoutOverall less than 100000 miliseconds. For example 50000 miliseconds by default, because the default HttpClient timeout equals 100000 miliseconds. If someone will create it's own HttpClient (not by AddJsonClient method provided by the library) and apply our default policies, than we will have both TimeoutOverall and HttpClient timeout equal to the same value and the result could be unpredictable.
2. I want to keep the same order of constants and parameters as it was before. I mean that TimeoutOverall is a replacement for HttpClientTimeout, I want to preserve this order. Just move this line before the TimeoutPerTry. It would be easier for our users to understand the changes in the version 2.0: replacement of one thing with another thing, all other stuff is the same.
3. You have typo in the name TimeoutOverallInMillicesons. Typo in the word Milliseconds.

Reply:
All done.